src/libs/auth.js

```diff
@@ -93,8 +93,9 @@ window.forceSignIn = async token => {
 
 authStore.subscribe(async (state, oldState) => {
   if (!oldState.isSignedIn && state.isSignedIn) {
+    const isTrustedEmail = _.includes(state.user.email.match(/@.*/g)[0], ['@broadinstitute.org', '@verily.com', '@channing.harvard.edu'])
```

Comment:
Nit: I'm pretty sure the `g` isn't needed. `.*` is greedy so I don't think it's possible for there to be more than one match. The more I think about it, the more I feel like it's better to omit the `g`, particularly in the presence of `[0]`.